examples/generative/stylegan.py

```diff
@@ -22,6 +22,12 @@
 ## Setup
 """
 
+"""
+### Install latest TFA and TFDS
+"""
+!pip install tensorflow_addons
```

Comment:
You should use a shell command block for this:

```
"""shell
pip install tensorflow_addons
"""
```

Reply:
Thanks, I hadn't noticed this syntax before but see it not in the contribution guide